### ISSUE TITLE
Bad links on https://beta.fec.gov/registration-and-reporting/file-paper/

### ISSUE THREAD

**Issue by jwchumley** [18F/openFEC-web-app#1030]
https://beta.fec.gov/registration-and-reporting/file-paper/

THe links under legal resources should be: (the last one is the only one that is already correct)

https://www.gpo.gov/fdsys/pkg/CFR-2011-title11-vol1/pdf/CFR-2011-title11-vol1-sec100-19.pdf
https://www.gpo.gov/fdsys/pkg/CFR-2012-title11-vol1/pdf/CFR-2012-title11-vol1-sec101-1.pdf
https://www.gpo.gov/fdsys/pkg/CFR-2012-title11-vol1/pdf/CFR-2012-title11-vol1-sec102-1.pdf
https://www.gpo.gov/fdsys/pkg/CFR-2011-title11-vol1/pdf/CFR-2011-title11-vol1-sec102-2.pdf

**Comment by jwchumley** [comment]
Same thing. On this page all the legal resources links have the same problem. 
 https://beta.fec.gov/registration-and-reporting/essentials-house-and-senate-candidates-and-committees/

https://www.gpo.gov/fdsys/granule/CFR-2014-title11-vol1/CFR-2014-title11-vol1-sec102-10
change to
https://www.gpo.gov/fdsys/pkg/CFR-2014-title11-vol1/pdf/CFR-2014-title11-vol1-sec102-10.pdf

**Comment by emileighoutlaw** [comment]
Thanks for noticing, Jeff. Looking into it now.

**Comment by emileighoutlaw** [comment]
Hey Jeff — Just had a chance to take a look at your hyperlinks, and they all look correct to me. They don't link directly to the PDF of the code, but rather to a table of contents page for each referenced section (aside from the last link, which GPO didn't have TOC front-matter for).

I chose this broader, table view for the legal resources because they give users the option to look at it as either a PDF or an XML:
<img width="534" alt="screen shot 2016-01-13 at 4 25 02 pm" src="https://cloud.githubusercontent.com/assets/10552702/12308269/6612c32e-ba12-11e5-8c9b-9128ba5f0166.png">

Additionally, this view allows users to scroll down the page and see the resource highlighted within the context of the CFR's other sections (for a more complete view of the law):
<img width="700" alt="screen shot 2016-01-13 at 4 25 08 pm" src="https://cloud.githubusercontent.com/assets/10552702/12308295/82b43a08-ba12-11e5-8623-6abc7f59a9ba.png">

This also makes it easier for them to navigate to other sections of the law, should they choose. Does that make sense?

**Comment by jwchumley** [comment]
This was reported by someone in RAD. Their comment was as follows:

"Citation links aren't consistent. some point to general page, some the 
actual citation. for example see..."  
https://beta.fec.gov/registration-and-reporting/file-paper/ "
In legal resources section, the link to 100.19, 101.1, and 102.1 do not go 
directly to the citation code (102.2 does). "  "

It seems from the comment that they would prefer to link directly to the 
section but in any case they should probably all behave the same way.

**Comment by emileighoutlaw** [comment]
I fixed the last link on the File by Paper page (https://beta.fec.gov/registration-and-reporting/file-paper/) so that it goes to the 11 CFR 102.2 title page, like all the other links (https://www.gpo.gov/fdsys/granule/CFR-2009-title11-vol1/CFR-2009-title11-vol1-sec102-2). 

I clicked through the rest of the pages and links— the one link on File by Paper was the only one I could find that was an outlier. The 52 USC ones link to http://uscode.house.gov/, but that's because GPO doesn't have them online. Were there any others you noticed that were inconsistent?

In sum, they should all be consistent now. 

@amypike @AmyKort, if you have a minute to read through the issue above, what are your thoughts about keeping the links as they are (to Tables of Contents with PDF and XML options) as opposed to linking directly to the PDF of the code?
